native_client/Makefile

```diff
@@ -31,7 +31,7 @@ endif
 
 OS := $(shell uname -s)
 CFLAGS  += -L${TFDIR}/bazel-bin/tensorflow -L${TFDIR}/bazel-bin/native_client $(EXTRA_CFLAGS)
-LDFLAGS += -ltensorflow -ldeepspeech -lc_speech_features -lkissfft $(EXTRA_LDFLAGS)
+LDFLAGS += -ltensorflow -ldeepspeech $(EXTRA_LDFLAGS)
```

Comment:
Did the command that included building //tensorflow:libtensorflow.so complete successfully? It doesn't move the result to the root of the binary directory unless you build it explicitly, even if it's depended on. If so, I guess I'll have to try multiple paths, but would appreciate the verification first.

(i.e. try `bazel build -c opt --copt=-mtune=native --copt=-march=native --copt=-O3 //tensorflow:libtensorflow.so` and see if it puts libtensorflow.so in the 'right' place)